src/docs/Contributing/SubmitChanges/DesignToolsConventions/index.mdx

```diff
@@ -3,73 +3,129 @@ title: 'Design tools conventions'
+
+> Here is a [tutorial](https://help.figma.com/article/54-constraints) to use resizing constraints in Figma.
+
+### 5. Layer styles and text styles have been applied on the different components
+
+We created layer and text styles to make the component creation process easier. Make sure to use it, the second time it will be easier that the first time 🤘. This will also make the update process way faster.
+
+### 6. Paddings and margins use magic unit values (1mu = 16px)
+
+Every element must use our magic unit standard which is 1 magic unit for 16 pixels.
+
+> Please read the [Magic Unit documentation](https://mozaic.adeo.cloud/Foundations/MagicUnit/) to find out more informations about that.
+
+### 7. Every patterns coming from other libraries are up to date
+
+Thank you for checking that every patterns are up to date (look after the “Component updates available” notifications in Figma 🤓).
+
+> Please read [Figma documentation](https://help.figma.com/article/314-accepting-updates-from-your-team-library) to find out more informations about that.
+
+### 8. Layers are well organized and follow our guidelines
+
+Layers should be organized in a top to bottom reading direction to make the override process easier. For example, the header needs to be on top of the layer list and the footer on the bottom of this list. That corresponds to **top to bottom and left to right**.
+
+### 9. I followed the logic of atomic design to build my component
+
+When an element is used twice, you need to create a master component that will be usable and repeated by its instances on other components. This will make the modification process faster when you need to update the component.
+
+You can download the [designer kit](https://github.com/adeo/design-system--styleguide/releases/download/v1.0.1-alpha.19/designer-kit.zip) to check how the components are built. Every Figma file we produce is built the same way.
+
 # Abstract
 
 > From a designer point of view, Mozaic is based on Sketch and Abstract. Abstract is a versioning software for design files and allows teams to collaborate with a modern workflow. Like a developer does in GitHub, you need to ask for a review before merging your branches. We wrote some rules to follow before clicking **_Request Review_** on Abstract.
 
-## 1. The symbols have been created following the [definition of Done for symbols](/Contributing/Designers/definitionOfDoneSymbols/index.md) page.
+## 1. The symbols have been created following the definition of Done for designers.
 
 We wrote some definition of done rules when you create some symbols in Sketch. Please read these instructions before submitting a review on Abstract. It will make the process way easier for us and we thank you for that 🤘.
```

Comment:
Can you add a link to the "definition of done for symbols in Sketch" ?

Reply:
@tiloyi  The definition is now inside the same page, at the beginning.
Can we create anchors here ?

Reply:
Yes I think you should create an anchor for that. This can be useful for readers.

Reply:
Do you know the command?